src/pages/[organizationSlug]/index.tsx

```diff
@@ -74,7 +74,7 @@ const Home: FunctionComponent<Props> = ({ projects }) => {
           New Project
         </Button>
       }
-      title={activeOrg?.name ?? "Loading..."}
+      title={activeOrg.name ?? "Loading..."}
```

Comment:
The nullish operator can be removed now.


```suggestion
      title={activeOrg.name}
```